### PR TITLE
docs: command to persist database

### PR DESCRIPTION
**Pull request by Juesto**
more details/as requested in #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ After modifying a `translation/source/*.xml` file, run:
 docker compose run --rm ui /lila/bin/trans-dump
 ```
 
+### Make the database persistent
+
+```bash
+docker cp lila-docker-mongodb-1:/data/db ./database
```
Comment (fitztrev) [inline README.md:114]: Can `docker compose cp` be used here instead?

Comment (fitztrev) [inline README.md:114]: If this command gets added to the docs, I think we should also add the `database` folder to be gitignored out-of-the-box.

**Review by fitztrev — CHANGES_REQUESTED** [review]
After this command is run, what's the process to have the mongodb container use this volume on future starts?

File: README.md (hunk 1 above)
Comment (Juesto) [reply README.md:114]: perhaps?

**Comment by Juesto** [comment]
> After this command is run, what's the process to have the mongodb container use this volume on future starts?

docker automatically adds the volume to docker-compose.yml when it executes the copy, i think. not sure

**Comment by fitztrev** [comment]
> > After this command is run, what's the process to have the mongodb container use this volume on future starts?
> 
> docker automatically adds the volume to docker-compose.yml when it executes the copy, i think. not sure

Can you test this and the `docker compose cp` and report back?

File: README.md (hunk 1 above)
Comment (Juesto) [reply README.md]: yes

**Comment by Juesto** [comment]
> > > After this command is run, what's the process to have the mongodb container use this volume on future starts?
> > 
> > 
> > docker automatically adds the volume to docker-compose.yml when it executes the copy, i think. not sure
> 
> Can you test this and the `docker compose cp` and report back?

see commit, maybe i should be less vague about it

**Pull request merged**